client/context.go

```diff
@@ -217,7 +217,10 @@ func (ctx Context) WithInterfaceRegistry(interfaceRegistry codectypes.InterfaceR
 
 // WithViper returns the context with Viper field
 func (ctx Context) WithViper() Context {
```

Comment:
I add Viper field to Client Context in order to use it later to read from `client.toml` file in UpdateClientContextFromClientConfig.